src/pages/Product/components/ProductInfo/ProductInfo.css

```diff
@@ -0,0 +1,84 @@
+.product-info .slider-container {
+    position: relative;
+    overflow: hidden;
+    width: 508px;
+    height: 100%;
+}
+
+.product-info button {
+    font-size: 16px;
+}
+
+.product-info .tab-buttons .button-care,.button-description {
+    margin-right: 36px;
+}
+
+.product-info .slider {
+    display: flex;
+    transition: transform 0.3s ease-in-out;
+}
+
+.product-info .slide {
+    flex: 0 0 100%;
+}
+
+.product-info img {
+    max-width: 100%;
+    height: 100%;
+}
+
+.product-info .slider-button {
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    background-color: transparent;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    font-size: 24px;
+}
+
+.product-info .prev-button {
+    left: 40px;
+}
+
+.product-info .next-button {
+    right: 40px;
+}
+
+.product-info .price {
+    font-size: 18px;
+    margin-top: 24px;
+    margin-bottom: 24px;
+}
+.product-info .add-cart {
+    padding: 6px 0;
+    font-weight: 400;
+    font-size: 16px;
+}
+
+.product-info {
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+}
+
+.product-info .info-right {
+    width: 50%;
+    padding-left: 17%;
+}
+
+.product-info .button-container {
```

Comment:
remove `.button-container` as div and try to leave only button `.add-cart `
and add `cursor: pointer`

Reply:
rdy